template/runtime/src/lib.rs

```diff
@@ -304,9 +305,13 @@ impl evm::Trait for Runtime {
 pub struct EthereumFindAuthor<F>(PhantomData<F>);
 impl<F: FindAuthor<u32>> FindAuthor<H160> for EthereumFindAuthor<F>
 {
-	fn find_author<'a, I>(_digests: I) -> Option<H160> where
+	fn find_author<'a, I>(digests: I) -> Option<H160> where
 		I: 'a + IntoIterator<Item=(ConsensusEngineId, &'a [u8])>
 	{
+		if let Some(author_index) = F::find_author(digests) {
```

Comment:
@sorpaas , Is there a way to detect in the runtime if the it is a manual-seal ?

Reply:
It's possible -- you can check whether the digests contain Aura-specific consensus ones. If not, then declare it is in manual seal.

What's the reason we need to do this, though?

Reply:
Thanks.
The main reason was to be able to support some kind of authoring even in manual-seal. That would also allow tests to verify the find_author is working as expected (instead of checking it returns none)